packages/zk-eddsa-ticket-pcd/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.js
```

Comment:
let's make sure we're not adding anything to the repo we shouldn't be adding. things we probably shouldn't add include:
- intermediate build outputs
- large downloadable files
- anything else that is not strictly necessary for a consumer of this package to make proper use of this pcd